lib/migration.js

```diff
@@ -60,7 +60,7 @@ module.exports = ({ sequelize, Sequelize, lockService, config = {} }) => {
             migrations.forEach(migration => {
               files += '\n\t- ' + migration.file;
             });
-            logger.info('Executed migration files:', files);
+            logger.info(`Executed migration files: ${JSON.stringify(files)}`);
```

Comment:
What about:
```suggestion
            logger.info('Executed migration files: %s', files);
```
Because `files` is not a JSON object, that's already a string value that we are supposed to have formatted, but moreover because we said that we should avoid `JSON.stringify()` (https://confluence.sierrawireless.com/display/AVENG/Javascript+Coding+Conventions#JavascriptCodingConventions-Logging)